dashboards/victoriametrics.json

```diff
@@ -250,17 +252,19 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "sum(vm_data_size_bytes{job=\"$job_storage\", type!=\"indexdb\"})",
+          "expr": "sum(vm_free_disk_space_bytes{job=~\"$job\", instance=~\"$instance\", path=\"/storage\"})",
```

Comment:
I think it would be better showing the minimum available disk space left, i.e. to use `min()` function instead of `sum()` function.
It would be great also adding `by (instance)` grouping for easier detection of instances with small disk space.

Reply:
Totally agree with `min`!

> It would be great also adding by (instance) grouping for easier detection of instances with small disk space.

This is a single-stat panel, we can show only one number here. But on `vmstorage` row we have free disk space by instance in more details.